apps/wallet/src/ui/app/hooks/deepbook/useGetEstimate.ts

```diff
@@ -318,5 +319,6 @@ export function useGetEstimate({
 			quoteBalance !== '0' &&
 			!!signer &&
 			!!activeAddress,
+		retry: ESTIMATE_RETRY_COUNT,
```

Comment:
maybe doublecheck that dryrun throws when the transaction fails and if not we might need to throw ourselves in order for retry to work

Reply:
checked, the dryrun will throw something as below

```
dataFromEstimateErrorObject TypeError: Invalid u64 value: 222929929292929288634845949329408. Expected value in range 0-18446744073709551615
    at BcsType.validate (bcs-type.ts:120:1)
    at BcsType.serialize (bcs-type.ts:42:1)
    at pure.u64 (pure.ts:11:1)
    at DeepBookClient.placeMarketOrder (client.ts:233:1)
    at async getPlaceMarketOrderTxn (useGetEstimate.ts:92:1)
    at async Object.queryFn (useGetEstimate.ts:160:1)
```

Reply:
Checked, error will be thrown as Error object 
ie:
```
dataFromEstimateErrorObject TypeError: Invalid u64 value: 222929929292929288634845949329408. Expected value in range 0-18446744073709551615
    at BcsType.validate (bcs-type.ts:120:1)
    at BcsType.serialize (bcs-type.ts:42:1)
    at pure.u64 (pure.ts:11:1)
    at DeepBookClient.placeMarketOrder (client.ts:233:1)
    at async getPlaceMarketOrderTxn (useGetEstimate.ts:92:1)
    at async Object.queryFn (useGetEstimate.ts:160:1)
    ```